src/test/resources/graphql-examples/deposits/delete/existing.graphql

```diff
@@ -0,0 +1,6 @@
+mutation {
+    deleteDeposits(input: {depositId: "00000000-0000-0000-0000-000000000005"}){
+        clientMutationId
+        depositId
+    }
+}
```

Comment:
newline at the end please, such that we don't get weird diffs in the future.
please do this for all these example files.